EmbedPlayer/5.6.0/searches.dcl

```diff
@@ -10,12 +10,12 @@ resource search digizuite_system_framework_search_embed_98_0 {
       <searchField id="security" securityType="ItemSecurityReadAccess" visible="false" operator="AND" />
     </searchFields>
     <valueFields OutputType="SQLXML">
-      <valueField id="thumb" fieldName="asset.urlAbsolut(${to_string(resource.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.desktop_video_thumb_50019.item_guid)})" />
-      <valueField id="thumbTablet" fieldName="asset.urlAbsolut(${to_string(resource.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.tablet_video_thumb_50020.item_guid)})" />
-      <valueField id="thumbMobile" fieldName="asset.urlAbsolut(${to_string(resource.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.mobile_video_thumb_50021.item_guid)})" />
-      <valueField id="desktopH264" fieldName="asset.urlAbsolut(${to_string(resource.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.desktop_h264_2048kbit_ffmpeg_50016.item_guid)})" />
-      <valueField id="tabletH264" fieldName="asset.urlAbsolut(${to_string(resource.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.tablet_h264_2048kbit_ffmpeg_50017.item_guid)})" />
-      <valueField id="mobileH264" fieldName="asset.urlAbsolut(${to_string(resource.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.mobile_h264_1024kbit_ffmpeg_50018.item_guid)})" />
+      <valueField id="thumb" fieldName="asset.urlAbsolut(${to_string(data.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.desktop_video_thumb_50019.item_guid)})" />
+      <valueField id="thumbTablet" fieldName="asset.urlAbsolut(${to_string(data.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.tablet_video_thumb_50020.item_guid)})" />
+      <valueField id="thumbMobile" fieldName="asset.urlAbsolut(${to_string(data.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.mobile_video_thumb_50021.item_guid)})" />
+      <valueField id="desktopH264" fieldName="asset.urlAbsolut(${to_string(data.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.desktop_h264_2048kbit_ffmpeg_50016.item_guid)})" />
+      <valueField id="tabletH264" fieldName="asset.urlAbsolut(${to_string(data.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.tablet_h264_2048kbit_ffmpeg_50017.item_guid)})" />
+      <valueField id="mobileH264" fieldName="asset.urlAbsolut(${to_string(data.destination.frontenddata_sm.item_guid)},${to_string(resource.media_format.mobile_h264_1024kbit_ffmpeg_50018.item_guid)})" />
```

Comment:
Aren't the referenced media formats only used here?
If they are not used in the `DC` layer, and we have this search in a separate `EmbedPlayer` layer, I think we should move these media formats into the `EmbedPlayer` layer, instead of making data references.

Note, I might be wrong about the media formats not being used in the `DC` layer, so please double check this.

Reply:
they are not used in the DC layer - the formats/transcodes are located in 
EmbedPlayer/5.6.0/formats.dcl and EmbedPlayer/5.6.0/transcodes.dcl

Reply:
Ah, my bad.